### ISSUE TITLE
Front Matter does not parse when have space after ","

### ISSUE THREAD

**Issue by chokchai** [esm7/obsidian-map-view#187]
this will not parse into links (have space in between)
```
---
location: [36.80803895853234, 139.71113385769755]
---
```

this will parse into links
```
---
location: [36.80803895853234,139.71113385769755]
---
```

Thank you.

<img width="372" alt="image" src="https://github.com/esm7/obsidian-map-view/assets/639340/692ab17d-c559-4011-a1b1-8f5c6fb98a6f">
<img width="349" alt="image" src="https://github.com/esm7/obsidian-map-view/assets/639340/8e1ddafb-bfdd-43f3-91d7-dae2dfdfc10e">

**Comment by ParkerRobb** [comment]
I've run into this too, but initially thought it inconsequential to report.

**Comment by esm7** [comment]
Should not happen anymore in the just-released v5.0.0.
